contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2024-03-21
+title: 'We just cut our base pricing by X, and are 5-10x cheaper than any major competitor'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: PostHog news
+tags:
+  - Product updates
+---
+
+## tl;dr
+
+* We have cut our pricing by X for everyone using our product analytics - this is our biggest ever price decrease!
+* We are now _by far_ the cheapest product analytics tool on the internet, with by far the broadest selection of features available of everyone, in our lowest pricing tier. We are 5-10x cheaper than the nearest competitor.
+* We are going to continue making price cuts, this is the start.
+* We have done this by creating a new lower tier for non-identified events - you can read the docs for how to set yourself up here.
```

Comment:
> non-identified events

Is there a clearer way to describe this? 

_We've done this by creating a new, lower tier for anonymized events_

Something around _identified users_ and _anonymous users_ seems the most clear. I get that technically-speaking it's at the event-level, but the implications (if I'm understanding correctly) are that there's nothing personally-identifiable on them, thus only designed to be used in aggregate.

Reply:
Edit: I realize all I changed was _non-identified_ to _anonymized_ but it's clearer in my mind what that means.